app/models/document.rb

```diff
@@ -132,6 +132,10 @@ def humanized_document_type
     document_type.underscore.tr("_", " ")
   end
 
+  def first_edition
+    editions.order(id: :asc).first
+  end
```

Comment:
I think we should consider adding a sort order to the [existing `Document#editions` association](https://github.com/alphagov/whitehall/blob/4e2a50caba4224ed16689f4fc08157dd4cb24504/app/models/document.rb#L13).

For example:

```ruby
has_many :editions,
         -> { order(created_at: :asc, id: :asc) },
         inverse_of: :document
```

In which case, it'll be safe to call `document.editions.first` and we won't need this `first_edition` method.

For what it's worth: I think editions are already implicitly ordered like this, so I wouldn't expect this to break any tests 🤞🏻

Reply:
@jyoung-gds I've just merged in #6717 so it should now be safe to use `document.editions.first` and drop this method.